### PR TITLE
pulseaudio: update OpenSSL + libalsa

### PR DESCRIPTION
**Pull request by Croydon**
- [x] I've read the [guidelines](https://github.com/conan-io/conan-center-index/blob/master/docs/how_to_add_packages.md) for contributing.
- [x] I've followed the [PEP8](https://www.python.org/dev/peps/pep-0008/) style guides for Python code in the recipes.
- [x] I've used the [latest](https://github.com/conan-io/conan/releases/latest) Conan client version.
- [x] I've tried at least one configuration locally with the
      [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by dmn-star — APPROVED** [review]

File: recipes/pulseaudio/all/conanfile.py [hunk 1]
```diff
@@ -62,15 +62,15 @@ def requirements(self):
...
         if self.options.with_fftw:
             self.requires("fftw/3.3.8")
         if self.options.with_x11:
             self.requires("xorg/system")
         if self.options.with_openssl:
-            self.requires("openssl/1.1.1h")   
+            self.requires("openssl/1.1.1i")
         # FIXME: enable when #2147 is merged
         # if self.options.with_dbus
         #     self.requires("dbus/1.12.16")
```
Comment (prince-chrismc) [inline recipes/pulseaudio/all/conanfile.py:76]: https://github.com/conan-io/conan-center-index/pull/3950

```suggestion
        # FIXME: enable when #3950 is merged
        # if self.options.with_dbus
        #     self.requires("dbus/1.12.16")
```

File: recipes/pulseaudio/all/conanfile.py [hunk 2]
```diff
@@ -84,7 +84,7 @@ def _configure_autotools(self):
         if not self._autotools:
             self._autotools = AutoToolsBuildEnvironment(self)
             args=[]
-            
+
             # FIXME: add dbus when #2147 is merged
```
Comment (prince-chrismc) [inline recipes/pulseaudio/all/conanfile.py:88]:
```suggestion
            # FIXME: add dbus when #3950 is merged
```

File: recipes/pulseaudio/all/conanfile.py (hunk 1 above)
Comment (prince-chrismc) [reply recipes/pulseaudio/all/conanfile.py:76]: There's two other occurrences outside of the diff

**Review by prince-chrismc — APPROVED** [review]

File: recipes/pulseaudio/all/conanfile.py (hunk 1 above)
Comment (Croydon) [reply recipes/pulseaudio/all/conanfile.py:76]: You are right, but I'm not going to invalid reviews for that change

**Review by intelligide — APPROVED** [review]

**Review by SSE4 — APPROVED** [review]

**Pull request merged**